src/styles/_datatable.scss

```diff
@@ -0,0 +1,15 @@
+.data-table {
```

Comment:
Are those necessary? Is bootstrap's default table style to have tables as small as possible?